pages.fr/common/df.md

```diff
@@ -1,7 +1,7 @@
 # df
 
 > Montre un aperçu de l'utilisation de l'espace disque.
-> Plus d'informations : <https://www.gnu.org/software/coreutils/df>.
+> Plus d'informations : <https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/df.html>.
```

Comment:
```suggestion
> Plus d'informations : <https://manned.org/df.1posix>.
```

Reply:
The space character after `Plus d'information` should be remove as well, right?